### DISCUSSION TITLE
Unable to start build

### DISCUSSION THREAD

**Discussion by strongui** [xdan/jodit#996] [Q&A] [answered]
I'm trying to fix a bug in the editor, and I'm following the setup in the guide, but when I do npm start, it just opens make.js and nothing happens.

**Comment by xdan** [comment]
```sh
nvm use
npm ci
npm start
```
?

**Reply by strongui** [reply]
yep, did all of that, is there a specific node or npm version required?

**Reply by xdan** [reply]
some error comes up?

**Reply by xdan** [reply]
nvm use should install 16.15.0

**Reply by strongui** [reply]
![image](https://github.com/xdan/jodit/assets/13644601/d412fab7-72af-4e4e-9693-59d193b44853)

I get this when I run nvm use. I'm currently on 14.11.0, do I need to go to 16?

**Reply by xdan** [reply]
nvm install

**Reply by strongui** [reply]
![image](https://github.com/xdan/jodit/assets/13644601/10386279-2ded-4709-b0ea-0f71d10f4ba7)
This is what I get. npm start still just opens make.js in my vs code

**Reply by xdan** [reply]
ok, just install 
nvm install 16.15.0
node -v - should show 16.15

**Reply by strongui** [reply]
![image](https://github.com/xdan/jodit/assets/13644601/97cfc5eb-9f1a-4414-a0b7-7c9592bc0d49)
Already did that.
![image](https://github.com/xdan/jodit/assets/13644601/eb28d27e-1f87-403a-a717-f81a8248c11a)

**Reply by xdan** [reply]
npm ci ?

**Comment by xdan** [comment]
I got a problem:
did you install `make` ?

https://stackoverflow.com/questions/32127524/how-to-install-and-use-make-in-windows

**Comment by xdan** [comment, marked as answer]
`npm start` runs through `make`, and on Windows there's usually no `make` installed — so the OS just opens `make.js` in your editor instead of running it. Install GNU Make (e.g. `choco install make`, or use WSL / Git Bash), then `nvm use && npm ci && npm start` works. That missing `make` is the whole issue.
